scout/server/blueprints/variants/controllers.py

```diff
@@ -83,6 +84,18 @@ def populate_force_show_unaffected_vars(institute_obj, form):
         form.show_unaffected.data = True
 
 
+def set_persistent_filters_choices(
```

Comment:
```suggestion
def get_and_set_persistent_filters_choices(
```

Reply:
Mm let's go with populate_ then, we have a few of those already!